Makefile

```diff
@@ -224,3 +224,7 @@ kubelet:
 .PHONY: kube-proxy
 kube-proxy:
 	KUBE_GIT_VERSION=$(KUBE-PROXY_GIT_VERSION) KUBE_BUILD_PLATFORMS=windows/amd64 make -C kube-proxy WHAT=cmd/kube-proxy
+
+.PHONY : containerd
+containerd:
+	GOOS=windows VERSION=$(shell cd containerd && git describe --match 'v[0-9]*' --dirty='.m' --always) make -C containerd WHAT=bin/containerd.exe
```

Comment:
For my own education, what does the `WHAT` directive do?
Is this the equivalent of what we do in the [midstream dockerfile](https://gitlab.cee.redhat.com/openshift-winc-midstream/openshift-winc-midstream/-/blob/rhaos-4.13-rhel-8/distgit/containers/windows-machine-config-operator/Dockerfile#L36) i.e. just building container.exe (not building `ctr.exe` and `containerd-stress.exe` anymore)?

Reply:
Yes that's the intention WHAT directive builds from the directory it is pointed to, I realize it's not needed in this case, updating.